operator/v1/types_network.go

```diff
@@ -267,6 +267,11 @@ type OVNKubernetesConfig struct {
 	// +kubebuilder:validation:Minimum=0
 	// +optional
 	MTU *uint32 `json:"mtu,omitempty"`
+	// geneve port is the UDP port to be used by geneve encapulation.
+	// Default is 6081 (OpenShift default 9018)
```

Comment:
What does `Default is 6081 (OpenShift default 9018)` mean?

Reply:
The standard geneve port for usual users of geneve is UDP 6081. OpenShift moves ports, including geneve port to the range 9000-9999. We have chosen 9081. (I will fix the typo in the comment)

Reply:
@pecameron in this case, we don't - the default is still 6081.

Reply:
Changed.

Reply:
Changed min to 1
@danwinship @squeed @dcbw PTAL